spec/formatting.md

```diff
@@ -366,17 +372,27 @@ The order of _options_ MUST NOT be significant.
 
 For each _option_:
 
-- Resolve the _identifier_ of the _option_.
-- If the _option_'s right-hand side successfully resolves to a value,
-   bind the _identifier_ of the _option_ to the _resolved value_ in the mapping.
-- Otherwise, bind the _identifier_ of the _option_ to an unresolved value in the mapping.
-   Implementations MAY later remove this value before calling the _function_.
-   (Note that an _Unresolved Variable_ error will have been emitted.)
+1. Let `res` be a new empty mapping.
+1. For each _option_:
+   1. Let `id` be the string value of the _identifier_ of the _option_.
+   1. Let `rv` be the _resolved value_ of the _option_ value.
+   1. If `rv` is a _fallback value_:
+      1. Emit a _Bad Option_ error.
```

Comment:
This smells like a fatal, even though (see @catamorphism's comments below) we later say "MAY". Perhaps:

```suggestion
      1. If supported, emit a _Bad Option_ error.
```